doc/man7/ossl-guide-quic-client-block.pod

```diff
@@ -0,0 +1,292 @@
+=pod
+
+=begin comment
+
+NB: Changes to the source code samples in this file should also be reflected in
+demos/guide/quic-client-block.c
+
+=end comment
+
+=head1 NAME
+
+ossl-guide-quic-client-block
+- OpenSSL Guide: Writing a simple blocking QUIC client
+
+=head1 SIMPLE BLOCKING QUIC CLIENT EXAMPLE
+
+This page will present various source code samples demonstrating how to write
+a simple blocking QUIC client application which connects to a server, sends an
+HTTP/1.0 request to it, and reads back the response.
+
+We assume that you already have OpenSSL installed on your system; that you
+already have some fundamental understanding of OpenSSL concepts, TLS and QUIC
+(see L<crypto(7)>, L<ossl-guide-tls-introduction(7)> and
+L<ossl-guide-quic-introduction(7)>); and that you know how to
+write and build C code and link it against the libcrypto and libssl libraries
+that are provided by OpenSSL. It also assumes that you have a basic
+understanding of UDP/IP and sockets. The example code that we build in this
+tutorial will amend the blocking TLS client example that is covered in
+L<ossl-guide-tls-client-block(7)>. Only the differences between that client and
+this one will be discussed so we also assume that you have run through and
+understand that tutorial.
+
+For this tutorial our client will be using a single QUIC stream. A subsequent
+tutorial will discuss how to write a multi-stream client.
+
+The complete source code for this example blocking QUIC client is available in
+the B<demos/guide> directory of the OpenSSL source distribution in the file
+B<quic-client-block.c>. It is also available online at
+L<https://github.com/openssl/openssl/blob/master/demos/guide/quic-client-block.c>.
```

Comment:
Such link can get obsolete, do we really want it?

Reply:
The problem is I expect many people will read this without necessarily having access to the source download (e.g. because they are reading it from the website). I wanted to have an easy way for people to access it.

Reply:
OK